src/components/card/card.jsx

```diff
@@ -12,11 +12,23 @@ import * as styles from "./card.module.scss";
 const Card = ({ data }) => {
   const categoryBlog = useCategory(data.frontmatter.category);
 
+  const urlImages = () => {
+    if (typeof window !== 'undefined' && `${window.location.href}/` === window.location.origin) {
+      return data.frontmatter.thumbnail;
+    } else {
+      if (typeof window !== 'undefined') {
+        const imageUrl = window?.location?.origin;
+        return `${imageUrl}/${data.frontmatter.thumbnail}`
+      }
+    }
+    return typeof window !== 'undefined' && window.location.origin;
```

Comment:
this return a boolean in some case, is ok?

Reply:
I have changed the way I wrote the function so that I no longer need the return now